src/models/legislative-process.ts

```diff
@@ -0,0 +1,8 @@
+export interface proposingALaw {
```

Comment:
type and interface name should start with an uppercase as a general TypeScript convention.